src/views/Privacy/index.jsx

```
@@ -1,7 +1,165 @@
 import React from "react";
 
 const PrivacyPolicy = () => {
-  return <div>PrivacyPolicy</div>;
+  return (
+    <div className="p-6 md:p-12 font-sans text-gray-800 space-y-6">
+      <h1 className="text-3xl font-bold text-black">Privacy Policy</h1>
+
+      <p>
+        WotchGotcha ("we", "our", or "us") respects your privacy and is
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)